core/base/document.php

```diff
@@ -822,10 +822,8 @@ public function print_elements_with_wrapper( $elements_data = null ) {
 		}
 		?>
 		<div <?php echo Utils::render_html_attributes( $this->get_container_attributes() ); ?>>
-			<div class="elementor-inner">
-				<div class="elementor-section-wrap">
-					<?php $this->print_elements( $elements_data ); ?>
-				</div>
+			<div class="elementor-section-wrap">
```

Comment:
It should be removed also from Theme Builder.

Reply:
@matipojo its already been done: https://github.com/elementor/elementor-pro/pull/290#pullrequestreview-387806867